docs/knowledgebase/advanced/consensus.md

```diff
@@ -71,18 +74,17 @@ resolved by choosing the branch that has the most blocks built on it recursively
 
 ![GHOST rule](assets/consensus-ghost.png)
 
-## Block Production
+## Block Authoring
 
 Some nodes in a blockchain network are able to produce new blocks, a process known as authoring.
 Exactly which nodes may author blocks depends on which consensus engine you're using. In a
 centralized network, a single node might author all the blocks, whereas in a completely
-permissionless network, an algorithm must select the block author at each height.
+permissionless network, an algorithm must select the block author at each height. This iterative process confirms the integrity of the previous block.
```

Comment:
I don't think the addition is correct. The order is.
1. The node confirms the integrity of the previous block by re-executing it
2. The node runs the consensus algorithm, and if it is selected to author, it authors on top of the best block it confirmed to be valid.

Thinking about it more now, maybe you meant something like, "By authoring a block, the author attests that the parent block and all of its ancestors recursively were valid"?

Reply:
> Thinking about it more now, maybe you meant something like, "By authoring a block, the author attests that the parent block and all of its ancestors recursively were valid"?

Agreed. I don't think any new sentence is necessary here, but the "iterative process" one has to go.